src/org/zaproxy/zap/extension/ascan/ActiveScan.java

```diff
@@ -83,8 +84,8 @@
 	private static final Logger log = Logger.getLogger(ActiveScan.class);
 
 	public ActiveScan(String displayName, ScannerParam scannerParam, 
-			ConnectionParam param, ScanPolicy scanPolicy) {
-		super(scannerParam, param, scanPolicy);
+			ConnectionParam param, ScanPolicy scanPolicy, RuleConfigParam ruleConfigParam) {
+		super(scannerParam, param, scanPolicy, ruleConfigParam);

```

Comment:
Good question. I dont think we're using this anywhere else, but obviously I dont know about 3rd parties. Where do we draw the line? Do we just always maintain (and depreciate) the old interfaces just in case? I guess thats the safest option...